docs/vscode-extension/migrate-v1/migrate-v1-tab.md

```diff
@@ -0,0 +1,47 @@
+# Build UI-based Apps for Teams
```

Comment:
IMO, this doc is for those users who already migrated v1 tab projects to v2, thus no need to introduce tab again.
This doc supposes to focus on "how to local debug a migrated tab project", including prerequisites, manual steps, f5, ...